lib/modifications.rb

```diff
@@ -71,6 +71,29 @@ def self.modify!(results, random_seed = 3)
         end
       end
 
+      # add reaction.manifestation to allergy intolerance result because of must support changes in 3.1.1
+      already_contains_reaction_manifestation = false
```

Comment:
`.any?` will handle the early exit from the iteration.
```
already_contains_reaction_manifestation = 
  results.any? do |bundle|
    bundle
      .entry
      .select { |e| e.resource.is_a? FHIR::AllergyIntolerance }
      .map(&:resource)
      .any? do |resource|
        ...
      end
  end
```

Reply:
Nice, definitely cleaner. Updated to these changes.